README.md

```diff
@@ -1,13 +1,125 @@
+- To change the style; I liked the already set colors so I focused on making minor tweaks to the style and make it more responsive.
+- When adding the possibility to add a player name, this included a new feature where the first thing to appear on the game is a start new game screen; where the user sets their name and then can start the game.
+- Flipping cards was made possible by following this site description: [How to Flip a Card](https://www.w3schools.com/howto/howto_css_flip_card.asp)   
+  - The main thing learned from this was that I needed an inner wrapper for the two sides of the cards.
+  - Also that I managed to flip a card by just adding a css class to it. VERY DYNAMIC! 😃
+  
+  ```css
+  /* Whenever this class is added to a card it flips!! */
+  .card__inner--flipped {
+    transform: rotateY(180deg);
+    border: 2px dotted var(--secondary) !important;
+  }
+  ```
+
+### The Code clean-up
+As a last step I went through the code file by file and added descriptive comments and removed any debug comments.
+Like I did in the last project I used comment blocks to describe each function. I made a template for this last project and used that in this one as well.
+
+Template:
+```js
+/** <insert general function description>
+ * Logic:
+ * 1. <first logical step>
+ * 2. <second logical step>
+ * ... etc.
+ */
+```
+I do this mainly for my own learning purpose. By decribing a function I made using logic steps I learn alot about what I have done.
+<br>
+Also this helps any code reviewers follow along with my code, and maybe even teach the reviewer something 😃
+
+***
+
+## Reflections
+All in all this project was not filled with any major issues. I worked through the requirements with ease. However, no project is without issues...
+<br>
+- I struggled a bit in the beginning trying to follow along in the already set up code. Especially since I found some bugs in that code. So the preparation phase on day 1 did take longer than I had planned. But after chaging the structure to be more easy to read it went smoothly.
+- Since I decided to make the cards flippable (and CSS is not my strong suit), that process did take longer than any javascript process. But when I found an instructional I wanted to follow and get inspired by, I learned from that and found a dynamic way to apply the functionality into my project. 
+<br>
+
+If I were to continue on this project / start over I would:
+- I would add a custom confirm alert (since I already made a custom normal alert, I would probably add the same type of thing for the confirm action)
+- I would also probably work more on the style for the question selection.
+  - Maybe even make so that you can't guess the same thing twice.
+<br>
+
+***
 
 ## View it live
+https://guess-who-flips.netlify.app/
 
```

Comment:
Your ReadMe files are screaming "READ ME AGAIN AND AGAIN" 😻  and this is strange because I do not like to read 🙈